### PR TITLE
Feature/retain keys

### PR DESCRIPTION
**Pull request by arc1999**
Added retain keys flag which is only set when -r is also set.

### CODE REVIEW

File: cmd/hypervisor/commands/gen-config.go [hunk 1]
```diff
@@ -30,6 +34,8 @@ func init() {
 	genConfigCmd.Flags().BoolVarP(&replace, "replace", "r", false, replaceUsage)
 	genConfigCmd.Flags().VarP(&configLocType, "type", "m", configLocTypeUsage)
 	genConfigCmd.Flags().BoolVarP(&testEnv, "testing-environment", "t", false, testEnvUsage)
+	genConfigCmd.Flags().BoolVar(&retainKeys, "retain-keys", false, "retain current keys")
+
```
Comment (nkryuchkov) [inline cmd/hypervisor/commands/gen-config.go]: Could you please remove the redundant empty line?

File: cmd/hypervisor/commands/gen-config.go [hunk 2]
```diff
@@ -58,6 +64,27 @@ var genConfigCmd = &cobra.Command{
 		default:
 			log.Fatalln("invalid config type:", configLocType)
 		}
+		if replace && retainKeys && pathutil.Exists(output) {
+			if err := fillInOldKeys(output, &conf); err != nil {
+				log.Fatalln("Error retaining old key",err)
```
Comment (nkryuchkov) [inline cmd/hypervisor/commands/gen-config.go]: Could you please format this line (space after the comma) and change `key` to `keys`?

File: cmd/hypervisor/commands/gen-config.go [hunk 3]
```diff
@@ -58,6 +64,27 @@ var genConfigCmd = &cobra.Command{
...
+		return fmt.Errorf("error reading old config file: %w", err)
+	}
+
+	var oldConf hypervisor.Config
+	if err := json.Unmarshal(oldConfBytes, &oldConf); err != nil {
+		return fmt.Errorf("invalid old configuration file: %w", err)
+	}
+
+	conf.PK = oldConf.PK
+	conf.SK=oldConf.SK
```
Comment (nkryuchkov) [inline cmd/hypervisor/commands/gen-config.go]: Could you please add spaces around `=`?

File: cmd/hypervisor/commands/gen-config.go [hunk 4]
```diff
@@ -58,6 +64,27 @@ var genConfigCmd = &cobra.Command{
...
+
+	return nil
+}
```
Comment (nkryuchkov) [inline cmd/hypervisor/commands/gen-config.go]: Could you please add a new line at the end of the file?

**Review by nkryuchkov — CHANGES_REQUESTED** [review]
I've checked the feature, it worked fine. Good job! I have some requests regarding the changes in the PR:

1. Could you please change the target branch from `master` to `develop`? 

2. The PR description might look clearer if it was adjusted according to the template here: https://github.com/SkycoinProject/skywire-mainnet/blob/master/.github/pull_request_template.md

3. Please run `make format`.

**Review by nkryuchkov — APPROVED** [review]

**Review by jdknives — APPROVED** [review]
Good job. Next time please use the template for the PR description and mention the ticket associated with the PR.

**Pull request merged**